### PR TITLE
Add game description in game.conf

### PR DESCRIPTION
**Pull request by Avyukt1**

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
This should go in the readme.md unless description.txt has some special significance?

**Comment by appgurueu** [comment]
`description.txt` should not be used, use `game.conf` instead

**Comment by Avyukt1** [comment]
> `description.txt` should not be used, use `game.conf` instead

That doesn't work in MT. The `game.conf` only works in the ContentDB

**Comment by Panquesito7** [comment]
> `game.conf` only works in ContentDB

Nope. `game.conf` designed for both ContentDB and Minetest.
Also note that `description.txt` is deprecated:

- For games, use `game.conf` instead.
- For mods, use `mod.conf` instead.

So the description should be added in `game.conf` as @appgurueu said.

File: description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Use swords, guns, and grenades to combat the enemy and capture their flag before they capture yours.
```
Comment (Panquesito7) [inline description.txt:1]: I guess `description.txt` should be removed...pretty much nobody uses Minetest 0.4.x series now.

**Review by Panquesito7 — COMMENTED** [review]
PR title shall be changed to `Add game description in game.conf`.

**Pull request merged**